packaging/hudi-gcp-bundle/pom.xml

```diff
@@ -39,7 +39,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>25.1.0</version>
+        <version>26.15.0</version>
```

Comment:
Let's define this version in the root pom and use it in both `hudi-gcp` and `hudi-gcp-bundle` modules?

Reply:
sg. done. PTAL